lib/deployment_runner.py

```diff
@@ -117,8 +117,12 @@ def rhv_configuration(self, page):
 
         # Custom data center & cluster name in RHVSH disabled for QCI 1.0
         # See https://bugzilla.redhat.com/show_bug.cgi?id=1367777
-        if self.rhv.rhv_setup_type != 'selfhost':
+        # Temporarily disabled for RHV in QCI 1.1
+        # See https://bugzilla.redhat.com/show_bug.cgi?id=1413194
```

Comment:
I figured this would save us the trouble of re-enabling this functionality when the underlying issue with custom names is resolved in RHV and/or RHVSH.